gdcdictionary/schemas/project.yaml

```diff
@@ -155,3 +155,8 @@ properties:
   dbgap_accession_number:
     type: string
     description: "The dbgap accession number provided for the project."
+
+  last_manifest_indexing_release:
+    type: string
+    format: date-time
+    description: Track the last release date of the NCBI manifests
```

Comment:
could we make it something a bit more general that can be reused for other ETLs if needed?

maybe something like "last_submission_identifier": "Project-specific identifier for internal use, to track the last submission for this project"?